validation/valid/ttml2-valid-profile-combine-ignore.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- valid - ttp:profile - @combine 'ignore' -->
+<tt xml:lang="en" xmlns="http://www.w3.org/ns/ttml"
+  xmlns:ttva="http://skynav.com/ns/ttv/annotations" ttva:warnOn="all" ttva:expectedErrors="0" ttva:expectedWarnings="0" ttva:model="ttml2">
+  <head>
+    <profile xmlns="http://www.w3.org/ns/ttml#parameter" combine="ignore"/>
```

Comment:
Doesn't this test have to have at least two different profiles so that the processor behaviour can be verified as ignoring all but the first?

Reply:
These are validation processor tests, not profile semantics related processor tests. Here (as elsewhere under `validation/valid`, we are testing that a validation processor does not report a false negative for valid content.

Reply:
Seems like we need a presentation test too then.

Reply:
But not with this PR. Adding general profile processing semantic tests has a much broader scope as we don't do it at all yet (for either ttml1 or ttml2 testing). I don't disagree that it would be nice to add such tests, but we went out he door with both ttml1 and ttml2 without employing any such tests, so this would go beyond what we did for CR exits. I have added a separate issue #225 to do that (at an appropriate time, but did not mark a milestone).